tests/unit/test_extras.hxx

```diff
@@ -61,6 +65,7 @@ public:
     PE_XIND = 0;
     StaggerGrids = false;
     IncIntShear = false;
+    transform = std::unique_ptr<ParallelTransform>(new ParallelTransformIdentity());
```

Comment:
I'm not sure we want this line here in general as I think we'll want to test other transforms as well.

Reply:
Agreed. It doesn't look like we need a ParallelTransform for any of the tests so far, so it could just be left as `nullptr`, unless I've missed something?

Reply:
The ``interp_to()`` test does need a ParallelTransform.
``interp_to()`` calls ``fieldmesh->toFieldAligned(var)`` when interpolating in the y-direction.
``Mesh::communicate()`` calls ``getParallelTransform()``, and communicate is called by interp_to (unless region==RGN_NOBNDRY, which I have not set in the tests as it is not the default option, although maybe we should add a test with RGN_NOBNDRY as well).
Both of these make ``serial_tests`` segfault when calling ``setParallelTransform`` because they cannot read options.

Reply:
Adding `mesh->setParallelTransform(std::unique_ptr<ParallelTransform>(new ParallelTransformIdentity()));` to `SetUpTestCase` in test_interpolation.cxx should allow you to set the transform for the interpolation tests without hardcoding it for all tests that use FakeMesh (i.e. can remove the above line).

Reply:
Thanks @d7919! I've implemented now as you suggested.